### ISSUE TITLE
The mask used for testing 

### ISSUE THREAD

**Issue by cmyyy** [shepnerd/inpainting_gmcnn#25]
Hello, my question is  what does the mask used for testing on places2 , celebaHQ-256 looks like when you do quantitative experiment?

**Comment by shepnerd** [comment]
In our paper, we used rectangle masks in random locations for testing. The
released code and some pretrained models support the evaluation on the
stroke masks.

cmyyy <notifications@github.com> 于2019年7月18日周四 上午10:55写道：

**Comment by cmyyy** [comment]
So one 128*128 rectangle mask at random location of an image, and it is same for both the datasets,right?

**Comment by shepnerd** [comment]
Yes.

cmyyy <notifications@github.com> 于2019年7月18日周四 下午1:19写道：

**Comment by cmyyy** [comment]
And when do quantitative experiment on places2, do u randomly chose 2000 images? If so, shape will be different, and in the test.py 
"input_image_tf = tf.placeholder(dtype=tf.float32, shape=[1, config.img_shapes[0], config.img_shapes[1], 3])"
the shape is fixed, how do you cope with this situation?

**Comment by shepnerd** [comment]
Yes. 2k images are randomly chosen for evaluation since the full testing
set of places2 is huge. About the shape problem, L63-70 in test.py just
crop and resize the central part of the input images into the ones with the
target image size, which ensures the input images to the network are with
the same size.

cmyyy <notifications@github.com> 于2019年7月18日周四 下午1:58写道：

**Comment by cmyyy** [comment]
Another question,in the quanttative experiment, the places2 model was trained on random strokes or rectangle masks?

**Comment by shepnerd** [comment]
All experiments in the paper are conducted with rectangle masks.

cmyyy <notifications@github.com> 于2019年7月20日周六 下午3:46写道：

**Comment by cmyyy** [comment]
Could you provide the places2 pretrained model trained on rectangle masks?

**Comment by shepnerd** [comment]
https://github.com/shepnerd/inpainting_gmcnn/issues/20#issuecomment-506572261

**Comment by godchengzhihang** [comment]
> 
> 
> And when do quantitative experiment on places2, do u randomly chose 2000 images? If so, shape will be different, and in the test.py
> "input_image_tf = tf.placeholder(dtype=tf.float32, shape=[1, config.img_shapes[0], config.img_shapes[1], 3])"
> the shape is fixed, how do you cope with this situation?

Hello, is there a tensorflow version of the code for the quantitative analysis part of the paper?
